Akinator.Api.Net.Tests/ClientIntegrationTests.cs

```diff
@@ -1,561 +1,614 @@
-using Akinator.Api.Net.Enumerations;
-using Akinator.Api.Net.Model;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
-using System;
-using System.Linq;
-using System.Threading;
-using System.Threading.Tasks;
-
-namespace Akinator.Api.Net.Tests
-{
-    [TestClass]
-    public class ClientIntegrationTests
```

Comment:
Can you explain what exactly you did change in this file? Did you only add the new movie tests or did something else change? Unfortunately I cannot view the changes with the github comparison tool :(

Reply:
I only added movie tests, it looks like the GitHub diff viewer is acting up